schema/partner_show.js

```diff
@@ -131,6 +138,18 @@ const PartnerShowType = new GraphQLObjectType({
           .then(exclude(options.exclude, 'id'));
       },
     },
+    artworks_count: {
+      type: GraphQLInt,
+      args: {
+        artist_id: {
+          type: GraphQLString,
+          description: 'The slug or ID of an artist in the show.',
+        },
+      },
+      resolve: ({ id, partner }, options) => {
+        return total(`partner/${partner.id}/show/${id}/artworks`, options);
+      },
+    },
```

Comment:
It makes more sense to me to expose all the counts on a schema as it's own nested object—like: https://github.com/artsy/metaphysics/blob/11d272a8ee28a62364dfb6feea31bef21ca820c8/schema/artist/index.js#L105-L119

Also note the `numeral` field which exposes formatting options to the client and returns a String or Integer depending on whether or not you've specified a [format](https://github.com/artsy/metaphysics/blob/11d272a8ee28a62364dfb6feea31bef21ca820c8/schema/fields/numeral.js#L10)